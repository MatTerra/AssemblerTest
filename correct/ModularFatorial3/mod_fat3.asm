MOD_FIM: BEGIN
public n
public fim
public mod_fim
section data
n: space
SECTION TEXT
fim: output n
stop
end
