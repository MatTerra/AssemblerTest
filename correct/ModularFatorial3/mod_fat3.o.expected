H: mod_fat3
H: 4
H: 0100
H: D fim 0
H: D mod_fim 0
H: D n 3
T: 13 03 14 0
